doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -154,6 +154,7 @@ nRF9160: Asset Tracker v2
 
   * Use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.
   * The application now integrates the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.
+  * The application now the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
```

Comment:
Something missing here. "The application now (verb) the function ..."
Has available, can call or something like that?

Reply:
This is fixed in the other PR.  As noted, don’t review the first 2 commits

Reply:
Oh, sorry. I overlooked that comment.